### DISCUSSION TITLE
CipherOS-twelve-L sync issues

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#8657] [General]
https://cirrus-ci.com/task/4723158132457472

**Comment by sh3ikh-faisal** [comment]
> https://cirrus-ci.com/task/4723158132457472
Fix this in your local_manifest
error: Cannot checkout vendor_xiaomi_mido

**Reply by Alone0316** [reply]
> > https://cirrus-ci.com/task/4723158132457472
> > Fix this in your local_manifest
> > error: Cannot checkout vendor_xiaomi_mido

My manifest is correct i made CipherOS-twelve-L on 20 March 
here is the build log https://cirrus-ci.com/build/6664332757958656 see it completed successfully, i didn't change anything in my manifest https://github.com/Alone0316/local_manifest/tree/Cipher see my manifest was last updated in Jan
